--- a/PKGBUILD	2017-03-14 18:51:06.890515049 +0100
+++ b/PKGBUILD	2017-03-14 18:51:09.363885079 +0100
@@ -108,7 +108,7 @@
 
 prepare() {
   # Allow ccache to work
-  mv wine-patched-staging-$_pkgbasever $pkgname
+  mv wine-staging-$_pkgbasever $pkgname
 
   sed 's|OpenCL/opencl.h|CL/opencl.h|g' -i $pkgname/configure*
 
@@ -118,6 +118,12 @@
 }
 
 build() {
+
+  cpucount=$(( $(cat /proc/cpuinfo|grep processor|tail -1|awk '{print $3}') + 1 ))
+
+  echo "CPU count: $cpucount"
+  sleep 1
+
   cd "$srcdir"
 
   if [[ $CARCH == x86_64 ]]; then
@@ -133,7 +139,7 @@
       --enable-win64 \
       --with-xattr
 
-    make
+    make -j $cpucount
 
     _wine32opts=(
       --libdir=/usr/lib32
@@ -152,7 +158,7 @@
     --with-xattr \
     "${_wine32opts[@]}"
 
-  make
+  make -j $cpucount
 }
 
 package() {
